cdk/lib/__snapshots__/frontend.test.ts.snap

```diff
@@ -2,28 +2,6 @@
 
 exports[`The Frontend stack matches the snapshot 1`] = `
 Object {
-  "Mappings": Object {
-    "frontend": Object {
-      "CODE": Object {
-        "AlarmEnabledInProd": false,
-        "MembershipSubPromotions": "arn:aws:dynamodb:*:*:table/MembershipSub-Promotions-DEV",
-        "RedemptionCodes": "arn:aws:dynamodb:*:*:table/redemption-codes-DEV",
-        "domainName": "support.code.theguardian.com.origin.membership.guardianapis.com",
-        "hostedZoneId": "Z1E4V12LQGXFEC",
-        "maxInstances": 2,
-        "minInstances": 1,
-      },
-      "PROD": Object {
-        "AlarmEnabledInProd": true,
-        "MembershipSubPromotions": "arn:aws:dynamodb:*:*:table/MembershipSub-Promotions-PROD",
-        "RedemptionCodes": "arn:aws:dynamodb:*:*:table/redemption-codes-PROD",
-        "domainName": "support.theguardian.com.origin.membership.guardianapis.com",
-        "hostedZoneId": "Z1E4V12LQGXFEC",
-        "maxInstances": 6,
-        "minInstances": 3,
-      },
-    },
-  },
```

Comment:
We end up getting rid of all of these as now we have a separate CODE and PROD template we don't have to do any lookups